src/resolve/resolvectl.c

```diff
@@ -1402,6 +1402,56 @@ static int dump_list(Table *table, const char *prefix, char * const *l) {
         return 0;
 }
 
+static int strv_extend_extended_bool(char ***strv, const char *name, const char *value) {
+        if (STRPTR_IN_SET(value, "yes", "no"))
```

Comment:
parse_boolean?

Reply:
OK. We return "yes" or "no" over dbus, and changing that would be an unexpected incompatibility. But parse_boolean is indeed nicer.